Gemfile

```diff
@@ -38,6 +38,8 @@ gem 'gpx'
 gem "activeadmin", git: "https://github.com/activeadmin/activeadmin"
 gem "inherited_resources", git: "https://github.com/activeadmin/inherited_resources"
 gem "activerecord-import"
+gem "delayed_job_active_record"
```

Comment:
I first looked at Resque but after reading this https://github.com/resque/resque#resque-vs-delayedjob I decided we could use the simpler delayed_job as we weren't doing anything to complex.